dev/integration_tests/semantics/test_driver/android_testing.dart

```diff
@@ -0,0 +1,512 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:test/test.dart';
+
+
+/// ClassName constants used by Flutter semantics.
+class AndroidClassName {
+  /// A checkbox widget.
+  static const String checkBox = 'android.widget.CheckBox';
+
+  /// The default className if none is provided by flutter.
+  static const String view = 'android.view.View';
+
+  /// A radio widget.
+  static const String radio = 'android.widget.RadioButton';
+
+  /// An editable text widget.
+  static const String editText = 'android.widget.EditText';
+
+  /// A read-only text widget.
+  static const String textView = 'android.widget.TextView';
+}
+
+/// Action constants Taken from [AccessibilityAction].
+class AndroidSemanticsAction {
+  const AndroidSemanticsAction._(this.id);
+
+  /// The Android id of the action.
+  final int id;
+
+  @override
+  int get hashCode => id.hashCode;
+
+  @override
+  bool operator ==(Object other) {
+    if (other.runtimeType != runtimeType) 
+      return false;
+    final AndroidSemanticsAction typedOther = other;
+    return id == typedOther.id;
+  }
+
+  /// Taken from [AccessibilityAction.ACTION_FOCUS].
```

Comment:
Out of curiosity: Are these hand-curated or is there an automated way to generate this?

Reply:
I considered writing a script but I did the napkin math on time to test/develop versus ctrl+C/ctrl+V and decided against it.